buf.yaml

```diff
@@ -0,0 +1,16 @@
+build:
+  roots:
+    - .
+lint:
+  use:
+    - MINIMAL
+    - FILE_LOWER_SNAKE_CASE
+    - UNARY_RPC
+  except:
+    - PACKAGE_DIRECTORY_MATCH
+  ignore:
+    - third_party
+breaking:
+  use:
+    - FILE
```

Comment:
@marbar3778 `FILE` implies `PACKAGE`, so there's no need for both. See https://buf.build/docs/breaking-checkers#choosing-a-category